lib/latte/Install.py

```diff
@@ -21,77 +37,50 @@
                  or: python Install.py -m gfortran
                  or: python Install.py -v 1.2.1 -b
 
-specify one or more options, order does not matter
-
-  -b = download and build the LATTE library
-  -p = specify folder of existing LATTE installation
-  -m = copy Makefile.lammps.suffix to Makefile.lammps
-  -v = set version of LATTE library to download and set up (default = 1.2.1)
-
 Example:
 
 make lib-latte args="-b -m gfortran"   # download/build in lib/latte
 make lib-latte args="-p $HOME/latte"   # use existing LATTE installation
 """
 
-# settings
 
-version = '1.2.1'
+pgroup = parser.add_mutually_exclusive_group()
+pgroup.add_argument("-b", "--build", action="store_true",
+                    help="download and build the Eigen3 library")
+pgroup.add_argument("-p", "--path",
+                    help="specify folder of existing Eigen installation")
+parser.add_argument("-m", "--machine", choices=['gfortran','ifort','linalg','serial','mpi'],
```

Comment:
missing `default=suffix`

Reply:
Thanks, there should be no default, or else the suffixflag variable would always be true. However, suffix needs to be initialized, so that the final section works correctly, if no `Makefile.lammps` exists and a machine option is not provided.